x-pack/plugins/spaces/public/views/nav_control/components/spaces_menu.tsx

```diff
@@ -163,13 +170,20 @@ class SpacesMenuUI extends Component<Props, State> {
 
   private renderSpaceMenuItem = (space: Space): JSX.Element => {
     const icon = <SpaceAvatar space={space} size={'s'} />;
+    const itemAriaLabel = i18n.translate('xpack.spaces.headerNavigationSwitchAriaLabel', {
```

Comment:
Suggest changing message id to `xpack.spaces.navControl.spacesMenu.headerNavigationSwitchAriaLabel` to be consistent with others in this file.

FWIW, I prefer the shorter name you have, but I'd rather be consistent here, and deal with a wholesale rename in the future.